tests/suites/test_suite_pkcs7.function

```diff
@@ -26,6 +26,15 @@ int pkcs7_parse_buffer( unsigned char *pkcs7_buf, int buflen )
 }
 /* END_SUITE_HELPERS */
 
+/* BEGIN_CASE */
+void pkcs7_asn1_fail( data_t *pkcs7_buf ) {
```

Comment:
I would really like to see this testing for memory leaks, but unfortunately I'm not sure we have a good way to do this in our test code at the moment. I've manually verified that all of these tests perform the same number of `mbedtls_free` as `mbedtls_calloc`, i.e., do not appear to have an obvious memory leak.

Reply:
I have been using `valgrind ./test_suite_pkcs7`, it has identified a couple of issues during development. No errors now though.